compiler/test/runnable/closure.d

```diff
@@ -981,7 +985,7 @@ int main()
     test9685b();
     test12406();
     test14730();
-    test14730x();
+    //test14730x();
```

Comment:
Why this commented out? If this is related to what is right above the diff, add a comment being explicit about it.

Reply:
See line 925

Reply:
Why not just delete it altogether?